test/jdk/java/nio/file/Files/copy.policy

```diff
@@ -0,0 +1,4 @@
+grant {
```

Comment:
Hello Brian, the copyright header is missing on this new file.

Reply:
You are right, I hadn't noticed that. The ones I had referred to were all in `test/jdk/java/net/httpclient/security/` and they have the copyright headers.